sample/itembox/itembox_sample.json

```diff
@@ -0,0 +1,17 @@
+// Target-Tables: ESS_MS_Controls_ItemBox, ESS_MS_Controls_ItemBox_Profiles
+{
+  "id": "<unique-id>",
+  "name": "<internal-name>",
+  "title": "<localized-name/title>",
+  "description": "<localized-description>",
+  "profiles": [
+    {
+      "displayName": "<localized-profile-name>",
+      "type": "grid",
+      "grid": "<grid-name>",
+      "isActive": true,
+      "defaultSearchString": "%",
+      "regex": null
```

Comment:
this needs select statement in current table

Reply:
are itemboxes only used with grids? if so ignore the above.

Reply:
Yes, this is the Grid-Name now